src/contributing-development.md

```diff
@@ -1,5 +1,37 @@
 # Contributing / Development
 
-There are several angles that people can take to contribute to this effort.
+There are several ways that people can help `offchain::ipfs`.
 
-TODO: Enumerate them
+## Create a dApp or custom Substrate pallet, and tell us about it
+
+Now that the proof-of-concept is complete and this manual is published, our goal is to be
+laser-focused on user needs and to build out the capabilities of `offchain::ipfs` based on those.
+So, **if you end up trying this out, we would love to know how, and for what.**
+
+- Tell us about what you're doing by opening an information issue at [rs-ipfs/substrate]
+- Share feedback on this manual by opening an issue at [rs-ipfs/offchain-ipfs-manual]
+- Share feedback on `offchain::ipfs` itself by open an issue at [rs-ipfs/substrate]
+
+## Support `offchain::ipfs` financially
+
+[Equilibrium] is a consultancy that builds, and helps you build _on_, the distributed web. If
```

Comment:
```suggestion
[Equilibrium] is a consultancy that builds, and helps you build _on_ the distributed web. If
```
I know what the purpose of that extra comma was, but the sentence is pretty short, so it felt odd